contracts/utils/Address.sol

```diff
@@ -54,8 +54,10 @@ library Address {
      * plain `call` is an unsafe replacement for a function call: use this
      * function instead.
      *
-     * If `target` reverts with a revert reason, it is bubbled up by this
-     * function (like regular Solidity function calls).
+     * If `target` reverts with a revert reason or custom error, it is bubbled
+     * up by this function (like regular Solidity function calls). However, if
+     * the call reverted with no returned reason, it default revert with an
+     * {FailedInnerCall} error.
```

Comment:
```suggestion
     * the call reverted with no returned reason, this function reverts with a
     * {FailedInnerCall} error.
```

Reply:
Fixed in [3d09549](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4502/commits/3d09549bd3cd5110f78979e217d7e2d4e9cf2434)